litestar/_openapi/schema_generation/schema.py

```diff
@@ -592,7 +592,7 @@ def for_enum_field(
         schema.type = enum_type
         schema.enum = enum_values
         schema.title = get_name(field_definition.annotation)
-        schema.description = field_definition.annotation.__doc__
+        self.process_schema_result(field_definition, schema)
```

Comment:
Should add a test that this doens't change the default behaviour, and still uses the member docstring if nothing else is specified

Reply:
The behaviour is desired; It mirrors the functionality of how we process e.g. dataclass members.